pkg/catalogv2/content/content.go

```diff
@@ -421,3 +437,11 @@ func isHTTP(iconURL string) bool {
 	u, err := url.Parse(iconURL)
 	return err == nil && (u.Scheme == "http" || u.Scheme == "https")
 }
+
+// isRancherCatalog - checks if the current chart repo is from the default rancher official helm catalog
+func isRancherCatalog(name string) bool {
```

Comment:
We get to know if it is bundled package or not by the directory in which it is located not by name since the user can always edit change ClusterRepos. 

So, can we use something like this ?

```
dir := gitDir(namespace, name, repo.status.url)
return strings.HasPrefix(dir, staticDir) || strings.HasPrefix(dir, localDir)
```

Reply:
Placed a new strategy by transforming some private methods into public, please review it @rohitsakala